scripts/azureml-assets/azureml/assets/get_tokens.py

```diff
@@ -31,7 +31,15 @@ def get_tokens(input_dirs: List[Path],
     """
     json_info = defaultdict(dict)
 
-    # Filter to only models
+    # Generate SAS tokens for generic assets
+    for asset_config in util.find_assets(
```

Comment:
Please call `util.find_assets()` just once with all asset types to find and then handle generic vs. models inside the loop.

Reply:
Updated, thanks.